aptos-move/aptos-release-builder/src/components/feature_flags.rs

```diff
@@ -93,6 +93,7 @@ pub enum FeatureFlag {
     VMBinaryFormatV7,
     ResourceGroupsChargeAsSizeSum,
     CommissionChangeDelegationPool,
+    ProposeSystemTransactions,
```

Comment:
Is it better to put in OnChainConsensusConfig instead? @zekun000

Reply:
FeatureFlag makes more sense to me. It is not consensus specific feature.

Reply:
tbh this is more like a consensus feature whether to propose system txn

Reply:
No strong opinions